src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +15,10 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        User user = userService.findByEmail(email);
+        if (user.getPassword().equals(password) && user != null) {
```

Comment:
What happens if user will be `null`? Could we call any method from `null`? 
You should check first on `null` and after checking password equality
```suggestion
        if (user != null && user.getPassword().equals(password)) {
```